### PR TITLE
chore: cleanup studio dependencies

### PR DESCRIPTION
**Pull request by kevcodez**
- Get rid of unused dependencies
- Move dev dependencies to dev dependencies

### CODE REVIEW

File: apps/studio/package.json [hunk 1]
```diff
@@ -52,8 +49,6 @@
     "file-saver": "^2.0.5",
     "framer-motion": "^6.5.1",
     "generate-password": "^1.7.0",
-    "graphql": "^16.8.1",
-    "graphql-ws": "^5.14.1",
```
Comment (alaister) [inline apps/studio/package.json:56]: These might still be needed for GraphiQL

Comment (kevcodez) [reply apps/studio/package.json:56]: I've successfully triggered some graphl requests, what else should I test?

These might also be transitive dependencies from graphiql

Comment (kevcodez) [reply apps/studio/package.json:56]: Already a transitive dep, shouldn't be a need to define it separately

Comment (alaister) [reply apps/studio/package.json:56]: Noice!

**Review by alaister — APPROVED** [review]
🔥

(if only the red lines were green 😮‍💨)

**Pull request merged**